.github/workflows/tests-off.yml

```
@@ -0,0 +1,58 @@
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - BUILD_TYPE: Release
+            STANDALONE: ON
+
+          - BUILD_TYPE: Release
+            STANDALONE: OFF
+
+          - BUILD_TYPE: Debug
+            STANDALONE: ON
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)